### ISSUE TITLE
Disabling ldap/oauth when using docker

### ISSUE THREAD

**Issue by shrippen** [wekan/wekan#4547]
### Server Setup Information
* Did you test in newest Wekan?: Yes
* Did you configure root-url correctly so Wekan cards open correctly (see https://github.com/wekan/wekan/wiki/Settings)? Yes
* Operating System: Ubuntu
* Deployment Method (Snap/Docker/Sandstorm/bundle/source): Docker
* Http frontend if any (Caddy, Nginx, Apache, see config examples from Wekan GitHub wiki first): Caddy
* Node.js Version: Docker
* MongoDB Version: 5.0.8
* What webbrowser version are you using (Wekan should work on all modern browsers that support Javascript)? Edge Chromium

### Problem description
I have configured wekan to use ldap and oauth via the env variables. Now I have concluded my tests and want to remove them again from the installation. So I set the corresponding env to false. But that didn't make any difference. I concluded that the setting must have been saved somewhere to mongodb but couldn't find where the settings for ldap are saved (I could find Oauth).

So to summarize I expected the setting of the ldap and oauth usage setting to false to disable these possibilities.

**Comment by xet7** [comment]
users table/collection, per-user setting. You could also use Admin Panel to change from ldap to password login. But if there is a lot of users, maybe some database query to update all at once would be better.

There are also settings for default login method at Admin Panel.

**Comment by xet7** [comment]
Admin Panel / People / People

**Comment by shrippen** [comment]
This enabled me to restrict the users from using ldap to login. But when I delete the user manually via the admin panel the ldap cronjob readds them every minute. I can't seem to stop it via the env variables.

**Comment by xet7** [comment]
@shrippen 

Have you removed LDAP settings from your docker-compose.yml ?

**Comment by xet7** [comment]
At database there is also AccountSettings or other Settings table/collection, where could be some LDAP stuff.

**Comment by xet7** [comment]
At database also could be some cron that updates LDAP users

**Comment by xet7** [comment]
or at integrations

**Comment by xet7** [comment]
Well, any LDAP_ settings should be commented out, when not using LDAP. And not add some settings like LDAP_ENABLE=false.

**Comment by shrippen** [comment]
I did both without any effect on the cronjob.

**Comment by xet7** [comment]
WeKan LDAP code is somewhere here https://github.com/wekan/wekan/tree/master/packages

**Comment by xet7** [comment]
Here are those LDAP PRs:
https://github.com/wekan/wekan/pulls?q=is%3Apr+is%3Aclosed+LDAP

**Comment by xet7** [comment]
@shrippen 

Hmm, I don't know do you have possibility to subscribe to Commercial Support at https://wekan.team/commercial-support/ , to help fund time spent figuring this out and fixing it?

I also have experienced elsewhere that not everything works like I would expect.

**Comment by gramakri** [comment]
I maintain the Cloudron package which uses a different docker image than the upstream one.

The code in https://github.com/wekan/wekan/blob/df54863e7243b0b067ec2d30d8352ff1838931c4/packages/wekan-ldap/server/loginHandler.js#L35 correctly uses the fallback handler when ENABLE_LDAP is not "true". And same with the user syncing logic - https://github.com/wekan/wekan/blob/df54863e7243b0b067ec2d30d8352ff1838931c4/packages/wekan-ldap/server/syncUser.js#L16 . The env parsing is also seemingly fine - https://github.com/wekan/wekan/blob/df54863e7243b0b067ec2d30d8352ff1838931c4/packages/wekan-ldap/server/ldap.js#L47 . Not sure where the error is at then. In Cloudron package, we don't dynamically turn off ldap.

@shrippen these settings are not stored in mongodb.
